cs/src/Management/TunnelRequestOptions.cs

```diff
@@ -80,22 +80,19 @@ public class TunnelRequestOptions
         /// </summary>
         public bool RequireAllTags { get; set; }
 
-        /// <summary>
-        /// Gets or sets an optional list of scopes that should be authorized when retrieving a
-        /// tunnel or tunnel port object.
-        /// </summary>
-        /// <remarks>
-        /// For service-to-service calls using forwarded client credentials, the scope(s) should
-        /// match the action the client is requesting, such as "host" or "connect". This enables
-        /// the calling service to ensure the client is specifically authorized for the supplied
-        /// scope(s), rather than just any scopes permitted by the API.
-        /// </remarks>
-        public string[]? Scopes { get; set; }
```

Comment:
The `scopes` property is removed here (and in other languages) because it was always an internal-only parameter and now is no longer needed in the SDK since the relay service uses a different internal API for tunnel connections.